drivers/mtd/mtd_partition.c

```diff
@@ -149,7 +149,7 @@ static int     part_procfs_stat(FAR const char *relpath,
 #if defined(CONFIG_FS_PROCFS) && !defined(CONFIG_PROCFS_EXCLUDE_PARTITIONS)
 static struct mtd_partition_s *g_pfirstpartition = NULL;
 
-const struct procfs_operations part_procfsoperations =
+const struct procfs_operations g_part_operations =
```

Comment:
I really dislike long variable names.  However, they are often necessary if a variable has global scope.  It is a good coding style to append something that forces the name into a "namespace" and prevents naming collisions.

I would suggest not simplifying the names like this.  g_part_operation could very easily collide with some other variable naming in the future.  I recommend extending the name with a "namespce" like `g_procfs_part_operation`.

I understand that that is a long cumbersome name but we will be happier in the long run if we try to prevent naming collisions.

Reply:
This change follow others to make the style consistent with each other:
https://github.com/apache/nuttx/pull/9033/files#diff-4f09fc6787bdafa4c18947e2cd05b138fa7d8e5a66c446e45e7076256d2f98c7L55-L67
We can add back procfs_ prefix.